### DISCUSSION TITLE
episode/S01E00

### DISCUSSION THREAD

**Discussion by giscus** [opensource-f2f/website#42] [General]
# episode/S01E00

《开源面对面》是个系列访谈节目，旨在通过采访不同行业、岗位的开源参与者的亲身经历、心得体会，让更多的从业人员感受到参与开源是一种积极的趋势与良好的 职业发展方向，认识到任何人都可以参与开源，并在开源的世界里体现自己的价值。

https://opensource-f2f-website-git-linuxsuren-patch-1-open-source-f2f.vercel.app/episode/S01E00

**Comment by LinuxSuRen** [comment]
节目从第一期开始，也走了一年了，感谢各位的一路陪伴
https://github.com/opensource-f2f/episode/issues/153

**Comment by MiracleShadow** [comment]
希望可以增加Spotify播客订阅

**Reply by LinuxSuRen** [reply]
感谢反馈！

嗯，之前讨论过，但遇到一些技术问题，大致上是喜马拉雅的格式有问题导致的。现在可以考虑再解决下这个问题，比如：用 GitHub pages 把喜马拉雅的服务再包装一层。

https://github.com/opensource-f2f/episode/issues/18

**Reply by wey-gu** [reply]
感谢反馈，抱歉我挖的坑还没填，see https://github.com/opensource-f2f/episode/issues/18

**Reply by LinuxSuRen** [reply]
稍微改造下 https://github.com/opensource-f2f/open-podcasts 的话，应该能做这个事情。https://github.com/opensource-f2f/devops-talk 这个就是基于前者实现的。

**Reply by wey-gu** [reply]
update: 我发现现在喜马拉雅的 xml 里有 itunes:email 字段了，应该可以直接提交了，我试试哈。

**Reply by wey-gu** [reply]
@LinuxSuRen send me the code you got from your gmail now.
<img width="884" alt="Screenshot 2022-11-18 at 17 53 30" src="https://user-images.githubusercontent.com/1651790/202673675-406b4925-c466-4af6-8b82-9fad2642d8dc.png">

**Reply by wey-gu** [reply]
成功拉，要等几个小时生效

https://open.spotify.com/show/48Yeu1L5pTluk1AVc3ShU6

<img width="1206" alt="Screenshot 2022-11-18 at 18 09 40" src="https://user-images.githubusercontent.com/1651790/202677020-b4396572-1e7f-490c-af73-d0709b593453.png">
